src/south/OPCUA_HA/OPCUA_HA.class.js

```diff
@@ -199,10 +199,21 @@ class OPCUA_HA extends ProtocolHandler {
     Object.keys(logs).forEach((statusCode) => {
       switch (statusCode) {
         case StatusCodes.BadIndexRangeNoData: // No data exists for the requested time range or event filter.
-          this.logger.debug(`${logs[statusCode].description} (${statusCode}): ${logs[statusCode].affectedNodes.toString()}`)
+          if (logs[statusCode].affectedNodes.length > 10) {
```

Comment:
Where this `10` is coming from? It would be better to define a constant for it with a descriptive name.